android/tangram/build.gradle

```diff
@@ -14,44 +14,65 @@ version = VERSION_NAME
 apply from: 'versioning.gradle'
 
 android {
-  compileSdkVersion 24
-
-  /*
-   * Ubuntu can't run the aapt on 64 bit before installing this packages
-   *    sudo apt-get install lib32stdc++6
-   *    sudo apt-get install lib32z1
-   */
-
-  buildToolsVersion '24.0.2'
+  compileSdkVersion 25
+  buildToolsVersion '25.0.0'
 
   defaultConfig {
     minSdkVersion 15
-    targetSdkVersion 24
+    targetSdkVersion 25
     versionCode buildVersionCode()
     versionName VERSION_NAME
+    externalNativeBuild {
+      cmake {
+        targets 'tangram'
+        arguments '-DPLATFORM_TARGET=android',
+                  '-DANDROID_TOOLCHAIN=clang',
+                  '-DANDROID_STL=c++_shared'
+        cppFlags '-std=c++14',
```

Comment:
I suppose it couldn't hurt, though there are no runtime errors produced by using things from the `<atomic>` headers so it's unclear whether it is necessary.